py/plugins/dev-local-vector-store/pyproject.toml

```diff
@@ -0,0 +1,30 @@
+[project]
+authors = [{ name = "Google" }]
+classifiers = [
+  "Development Status :: 3 - Alpha",
+  "Environment :: Console",
+  "Environment :: Web Environment",
+  "Intended Audience :: Developers",
+  "Operating System :: OS Independent",
+  "License :: OSI Approved :: Apache Software License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: 3.13",
+  "Programming Language :: Python :: 3 :: Only",
+  "Topic :: Scientific/Engineering :: Artificial Intelligence",
+  "Topic :: Software Development :: Libraries",
+]
+dependencies = ["genkit", "pytest-mock", "pytest-asyncio"]
+description = "Genkit Dev Local Vector Store Plugin"
+license = { text = "Apache-2.0" }
+name = "genkit-dev-local-vector-store-plugin"
```

Comment:
rename this and related stuff (files, variables, methods, etc) to `vectorstore`

Reply:
> rename this and related stuff (files, variables, methods, etc) to `vectorstore`

name `vectorstore` does not fully reflect the purpose of this plugin. So maybe better name it `local-vectorstore` to prevent ambiguity?

Reply:
I meant replace `dev-local-vector-store` to `dev-local-vectorstore` (make it one word)